### ISSUE TITLE
Return empty value on error, not "Element not found"

### ISSUE THREAD

**Issue by fvant** [gr0und-s3ct0r/niet#30]
When an element is not present or empty, please return an empty string and not the actual error text.

Given the yaml

version:
  lab: 1.0.0
  dev:

I would like  both   niet version.dev   and  niet version.prod   to simply return an empty string.

The documentation gives a  'deal with errors' example but does that work ?

**Comment by 4383** [comment]
@fvant thanks for this issue.

Yep "deal with errors" example works!

I want to add an command line option that allow you to return empty string when an element is not present only (key not found).

When the key exist and the value are equal to empty then we will just return an empty value.

**Comment by 4383** [comment]
Fixed in the development version.
It will be available on pypi inside the version `1.4.0` when we publish this version, that is to say, as soon as possible.

Example of use:
```sh
```sh
$ echo '{"foo": "bar", "fizz": {"buzz": ["1", "2", "3"]}}' | niet fizz.gogo --silent # with silent mode
$ echo $?
1
$ echo '{"foo": "bar", "fizz": {"buzz": ["1", "2", "3"]}}' | niet fizz.gogo # without silent mode 
Element not found: fizz.gogo
$ echo $?
1
```

Have fun!

**Comment by 4383** [comment]
Well! Now Available on [niet 1.4.0](https://pypi.org/project/niet/1.4.0/)
This feature is now available on pypi!
You can get it by updating niet:
```sh
pip install -U niet
```
Or by using the version `1.4.0`:
```sh
pip install niet==1.4.0
```
